doc/classes/RenderingDevice.xml

```diff
@@ -636,7 +636,7 @@
 			<param index="0" name="shader_source" type="RDShaderSource" />
 			<param index="1" name="allow_cache" type="bool" default="true" />
 			<description>
-				Compiles a SPIR-V from the shader source code in [param shader_source] and returns the SPIR-V as a [RDShaderSPIRV]. This intermediate language shader is portable across different GPU models and driver versions, but cannot be run directly by GPUs until compiled into a binary shader using [method shader_compile_binary_from_spirv].
+				Compiles an SPIR-V from the shader source code in [param shader_source] and returns the SPIR-V as a [RDShaderSPIRV]. This intermediate language shader is portable across different GPU models and driver versions, but cannot be run directly by GPUs until compiled into a binary shader using [method shader_compile_binary_from_spirv].
```

Comment:
I think it's pronounced "spurr - vee" and not spelled out letter by letter, hence using `a`. That's debatable I guess.

Reply:
Oh, that's odd, will restore

Reply:
Some "evidence":
http://casual-effects.blogspot.com/2016/03/computational-graphics-pronunciation.html
https://twitter.com/lunasorcery/status/1100506456732942336